content/en/users/aai/check-in/vos/_index.md

```diff
@@ -284,40 +284,238 @@ From the steps defined [above](#managing-cou-admin-members):
 
 ## VO membership API
 
+### API v2
+
+Check-in provides a REST API that allows clients to manage membership
+information only for the VOs they are authoritative for.
+
+Features:
+
+- Members of the VO are identified via their EGI Check-in Community User Identifier (CUID)
+- Membership can be limited to a specified period
+- All [REFEDS](https://wiki.refeds.org/display/STAN/eduPerson+2020-01#eduPerson202001-eduPersonAffiliation) membership affiliations are supported
+- Role titles are supported
+- Different membership status values are supported, namely `Active`, `Expired`,
+  `Deleted`, `Suspended`
+- Check-in automatically changes the membership status from `Active` to
+  `Expired` beyond the validity period
+
+#### Connection Parameters
+
+{{< tabpane >}}
+  {{< tab header="Production environment" lang="C" >}}
+  # Export VO API Base URL parameter
+  $ export VO_API_BASE_URL=https://aai.egi.eu/api/v2/VoMembers
+  # Export CO ID parameter.
+  # The CO ID is the number part of the API username prefix. e.g. for the username `co_2.test`, the CO_ID is `2`
+  $ export CO_ID=2
+  {{< /tab >}}
+  {{< tab header="Demo environment" lang="C" >}}
+  # Export VO API Base URL parameter
+  $ export VO_API_BASE_URL=https://aai-demo.egi.eu/api/v2/VoMembers
+  # Export CO ID parameter.
+  # The CO ID is the number part of the API username prefix. e.g. for the username `co_2.test`, the CO_ID is `2`
+  $ export CO_ID=2
+  {{< /tab >}}
+  {{< tab header="Development environment" lang="C" >}}
+  # Export VO API Base URL parameter
+  $ export VO_API_BASE_URL=https://aai-dev.egi.eu/api/v2/VoMembers
+  # Export CO ID parameter.
+  # The CO ID is the number part of the API username prefix. e.g. for the username `co_2.test`, the CO_ID is `2`
+  $ export CO_ID=2
+  {{< /tab >}}
+{{< /tabpane >}}
```

Comment:
Here you need to do:
* remove the spaces at the beginning
* add the prettier exclusion so that it won't be messed up with by prettier
* you should also specify that now it's shell language
* please also split the very long comments, like just before the `e.g.`
 
I started to do it in your PR but as you are updating the PR using forced push you deleted the changes I made.
 
`tabpane` usage got updated via #416, please see example at  https://github.com/EGI-Federation/documentation/blob/main/content/en/about/contributing/shortcodes/_index.md?plain=1#L145-L172

I expect this will fix most of the linting errors

Reply:
Done

Reply:
OK so the linter is really unhappy with this, let's ignore this and not having it a blocker.